.github/workflows/ci.yml

```diff
@@ -3,7 +3,32 @@ name: Run unit tests & check conventions
 on: [push, pull_request]
 
 jobs:
-  tests:
+  file-conventions:
+    runs-on: ubuntu-22.04
+    container:
+      image: "ubuntu:22.04"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install required dependencies
+        run: |
+          apt update
+          apt install -y sudo
+          # We need to install curl otherwise we get these errors in the CI:
+          # Unable to load the service index for source https://api.nuget.org/v3/index.json.
+          # The SSL connection could not be established, see inner exception. 
+          # The remote certificate is invalid because of errors in the certificate chain: UntrustedRoot 
+          sudo apt install -y curl
+
+      - name: Setup .NET
+        run: |
+          apt install -y dotnet6
+          dotnet dev-certs https --trust
```

Comment:
@realmarv why is this here? I thought it wasn't needed if curl is being installed